drivers/wdm/audio/hdaudbus_new/adsp.cpp

```diff
@@ -378,8 +378,8 @@ ADSP_BUS_INTERFACE ADSP_BusInterface(PVOID Context) {
 	busInterface.Size = sizeof(ADSP_BUS_INTERFACE);
 	busInterface.Version = 1;
 	busInterface.Context = Context;
-	busInterface.InterfaceReference = WdfDeviceInterfaceReferenceNoOp;
-	busInterface.InterfaceDereference = WdfDeviceInterfaceDereferenceNoOp;
+	busInterface.InterfaceReference = (PINTERFACE_REFERENCE)WdfDeviceInterfaceReferenceNoOp;
+	busInterface.InterfaceDereference = (PINTERFACE_DEREFERENCE)WdfDeviceInterfaceDereferenceNoOp;
```

Comment:
Ideally these casts shouldn't be needed. Could you reverify whether they still are? (iirc @DarkFire01 fixed these functions to correctly have the NTAPI)